packages/cli/src/commands/generate/cell/cell.js

```diff
@@ -141,9 +138,7 @@ export const files = async ({
   //    "path/to/fileB": "<<<template>>>",
   // }
   return files.reduce((acc, [outputPath, content]) => {
-    const template = generateTypescript
-      ? content
-      : transformTSToJS(outputPath, content)
+    const template = typescript ? content : transformTSToJS(outputPath, content)
```

Comment:
I think this will break with the new function signature.

Reply:
The function who's signature changed is from a different file, this one didn't change! See line 243. I just renamed the `generateTypescript` var to `typescript` to be consistent with the other generators.